src/_locales/en.config

```diff
@@ -150,6 +150,18 @@ Default
 Change the browser theme
 
 
+#==== Context menu ====
```

Comment:
Lets currently remove it, because it requires much work with all the locales.

The design will change and many messages will totally disappear, and some will appear. So we will start another campaign to ask people translate.